torch/csrc/distributed/rpc/rref_context.cpp

```diff
@@ -157,9 +159,12 @@ void RRefContext::delUser(
     const ForkId& forkId) {
   std::lock_guard<std::mutex> lock(destroyedMutex_);
   if (!destroyed_) {
-    auto fm = agent_->send(
+    rpc::RpcRetryOptions options;
+    options.maxRetries = 20;
+    auto fm = agent_->sendWithRetries(
         agent_->getWorkerInfo(owner),
-        RRefUserDelete(rrefId, forkId).toMessage());
+        RRefUserDelete(rrefId, forkId).toMessage(),
+        options);
```

Comment:
Is this idempotent? If so, can you add a detailed comment here why it is idempotent?